lib/condenser/processors/node_processor.rb

```diff
@@ -8,6 +8,10 @@ class NodeProcessor
     
     def self.setup(environment)
     end
+  
+    def name
+      self.class.name
+    end
```

Comment:
Awesome, can you remove this function from the other processors that inherit from `NodeProcessor` and duplicate this method, then this is good to merge

Reply:
@malomalo there are not anymore uses of `def name` on the processors